src/app/page.tsx

```diff
@@ -1,3 +1,4 @@
+"use client"
 import { Button } from "@/components/LandingPage/Button";
```

Comment:
we can not make the page.tsx as a client component, as next js says it is a server side and should kept as server side only.

Reply:
This can be resloved!

Reply:
> we can not make the page.tsx as a client component, as next js says it is a server side and should kept as server side only.

Actually I am using usestate and useeffect hooks in this file. So it need to be a client component as server components doesn't allow it.

Reply:
yaa, so in that case u should consider creating an another client component, and then should use here!

Reply:
Yeah, it can be done but need to shift all the content of page.tsx to a separate page as different components are using the theme. There should be only on componentin page.tsx like this:-
![image](https://github.com/SnipSavvy/SnipSavvy_Frontend/assets/151373021/c1c3633f-e430-4d23-95db-3c621d201e5a)

Will it create problem for you?